### ISSUE TITLE
merge Clr6 histone deacetylase complex I'' (GO:1990483) AND Rpd3L complex

### ISSUE THREAD

**Issue by ValWood** [pombase/curation#2512]
I think these should merge, but check annotations once corrected

4 | (Clr6 histone deacetylase complex I'' (GO:1990483) AND Rpd3L complex (GO:0033698))

**Comment by ValWood** [comment]
Rpd3L complex (GO:0033698) has 

Clr6 histone deacetylase complex II' as an exact synonym

**Comment by ValWood** [comment]
asked winston lab

**Comment by ValWood** [comment]
Rpd3L complex (GO:0033698)  needs pst3

remove
SPCC1682.13 | laf2 | Clr6 associated factor 2, Laf2
SPAC14C4.12c | laf1 | Clr6 L associated factor 1 Laf1
SPBC1734.16c | pst3 | SIN3 family co-repressor Pst3
from Rpd3S Rpd3S/Clr6-CII complex (GO:0032221)

**Comment by ValWood** [comment]
Clr6 related complexes are still mixed up
![Uploading Quick or new starter fix Clr6 complexes.png…]()

**Comment by ValWood** [comment]
<img width="767" alt="Quick or new starter fix Clr6 complexes" src="https://github.com/pombase/curation/assets/7359272/c7ab2157-64f2-44b0-90bc-5f01580edf3b">

**Comment by ValWood** [comment]
I've always been a bit confused by these alternative names for clr6 complexes. I think they are largely  resolved, do you agree?

**Comment by PCarme** [comment]
Yes, I recall we have resolved these alternative complexes last year

**Comment by ValWood** [comment]
great!
